website/docs/reference/resource-configs/tags.md

```diff
@@ -164,10 +200,65 @@ seeds:
 
 </File>
 
+### Apply tags to saved queries
+
+<VersionBlock lastVersion="1.8">
```

Comment:
Hiya @mirnawong1 

This is non-blocking suggestion. Would this benefit from an info callout? It may encourage the user to upgrade?

Kind Regards
Natalie